cli/src/wallet_cli/mod.rs

```diff
@@ -66,19 +66,19 @@ pub enum WalletCommand {
     /// Burn an amount of native token.
     BurnNativeToken {
         /// Token ID to be burnt, e.g. 0x087d205988b733d97fb145ae340e27a8b19554d1ceee64574d7e5ff66c45f69e7a0100000000.
-        token_id: String,
+        token_id: TokenId,
         /// Amount to be burnt, e.g. 100.
-        amount: String,
+        amount: U256,
```

Comment:
Is this really working as expected? If I remember correctly by default it didn't use `::from_dec_str()` by default but something else like hex or so which then ended up in wrong values

Reply:
Btw would be nice to have an own PR for such changes since this is not related to delegation

Reply:
I made one :)